jobs/scheduler/spec

```diff
@@ -32,7 +32,7 @@ properties:
   scheduler.tls.client_scalingengine.cert:
```

Comment:
Will fix the names for scheduler. 
For other components will do rename  in a separate story along with component specific CA.